package.json

```diff
@@ -265,13 +266,11 @@
     "jest-environment-jsdom": "^29.4.1",
     "jest-transformer-svg": "^2.0.1",
     "memfs": "^4.6.0",
-    "metro-react-native-babel-preset": "0.76.8",
     "onchange": "^7.1.0",
     "portfinder": "^1.0.28",
     "prettier": "^2.8.8",
     "pusher-js-mock": "^0.3.3",
     "react-native-clean-project": "^4.0.0-alpha4.0",
-    "react-native-flipper": "https://gitpkg.now.sh/facebook/flipper/react-native/react-native-flipper?9cacc9b59402550eae866e0e81e5f0c2f8203e6b",
```

Comment:
It seems the flipper plugin used for debugging was removed. There's also this warning during development

<img width="521" alt="Screenshot 2024-01-15 at 15 33 35" src="https://github.com/Expensify/App/assets/12156624/a0dc9142-f78c-4cd5-a188-3c9766d43b09">

Perhaps this was a mistake because Flipper is still being used in the project

Could you please clarify if the removal of the Flipper plugin was intentional and, if so, provide the reasoning behind this decision?

Reply:
This was dicussed here, not a blocker, Flipper will be removed https://expensify.slack.com/archives/C05LX9D6E07/p1705066075250729?thread_ts=1704921323.369379&cid=C05LX9D6E07